doc/book/admin/upgrades.rst

```diff
@@ -58,95 +50,78 @@ How to upgrade from Tarantool 1.7 to 2.x
-How to upgrade from Tarantool 1.6 to 1.7
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrading Tarantool in a replica set
+------------------------------------
 
-This procedure is for upgrading a standalone Tarantool instance in production
-from 1.6.x to 1.7.x. Notice that this will **always imply a downtime**.
-To upgrade **without downtime**, you need several Tarantool servers running in a
-replication cluster (see :ref:`below <admin-upgrades_replication_cluster>`).
+Below are the general instructions for upgrading Tarantool in a replica set.
+Upgrading from some versions can involve certain specifics. You can find
+instructions for individual versions :ref:`in the list below <admin-upgrades_version_specifics>`.
 
-Tarantool 1.7 has an incompatible :ref:`.snap <internals-snapshot>` and
-:ref:`.xlog <internals-wal>` file format: 1.6 files are
-supported during upgrade, but you won’t be able to return to 1.6 after running
-under 1.7 for a while. It also renames a few configuration parameters, but old
-parameters are supported. The full list of breaking changes is available in
-`release notes for Tarantool 1.7 <https://github.com/tarantool/tarantool/releases>`_.
+..  important::
 
-1. Check with application developers whether application files need to be
-   updated due to incompatible changes (see
-   `1.7 release notes <https://github.com/tarantool/tarantool/releases>`_).
-   If yes, back up the old application files.
+    The only way to upgrade Tarantool from 1.6 to 2.x **without downtime** is
+    taking an intermediate step by upgrading from 1.6 to 1.10 and then to 2.x.
+    This also applies to upgrading Tarantool from 1.7 and 1.9.
 
-2. Stop the Tarantool server.
+    Before upgrading Tarantool from 1.x to 2.y, please read about the associated
+    :ref:`caveats <admin-upgrades-1.6-1.10>`.
 
-3. Make a copy of all data (see an appropriate hot backup procedure in
-   :ref:`Backups <admin-backups>`) and the package from which the current (old)
-   version was installed (for rollback purposes).
-
-4. Update the Tarantool server. See installation instructions at Tarantool
-   `download page <http://tarantool.org/download.html>`_.
+Before the upgrade
+~~~~~~~~~~~~~~~~~~
 
-5. Update the Tarantool database. Put the request ``box.schema.upgrade()``
-   inside a :doc:`box.once() </reference/reference_lua/box_once>` function in your Tarantool
-   :ref:`initialization file <index-init_label>`.
-   On startup, this will create new system spaces, update data type names (e.g.
-   num -> unsigned, str -> string) and options in Tarantool system spaces.
+#. Check the replica set health by running the following code on every replica:
 
-6. Update application files, if needed.
+   ..  code-block:: tarantoolsession
 
-7. Launch the updated Tarantool server using ``tarantoolctl`` or ``systemctl``.
+       box.info.ro -- shouldn't be "orphan"
+       box.info.ro_reason -- since 2.10, shouldn't be "orphan"
+       box.info.status -- should be "running"
```

Comment:
Actually we should only check that `box.info.status` is "running" on all the instances and that at least one instance has `box.info.ro` == `false`.

In case all the instances have `box.info.ro == true`, one may check `box.info.ro_reason`, starting from version 2.10, to have a clue what the issue is